### PR TITLE
Nodes, Versions, Users

### PR DESCRIPTION
**Pull request by Baggerone**

### CODE REVIEW

File: domain_test.go [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+type testMACAddr struct {
+	MA1         string
+	MA2         string
+	ExpectError bool
+}
+
+func TestCleanMACAddress(t *testing.T) {
+
+	testMAs := []testMACAddr{
```
Comment (fillup) [inline domain_test.go]: I guess it is cool to support multiple formats, but I've only ever seen it reported as separated by colons and we don't have a test for that. Can you add some tests with data that looks like what we'll actually be receiving? ;-)

File: node/main.go [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+	"github.com/aws/aws-lambda-go/lambda"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+	"github.com/silinternational/speed-snitch-admin-api"
+	"github.com/silinternational/speed-snitch-admin-api/db"
+	"net/http"
+)
+
+var dynamo = dynamodb.New(session.New(), aws.NewConfig().WithRegion("us-east-1"))
```
Comment (fillup) [inline node/main.go]: this is unused right? probably a carry over before some `db/common.go` refactoring

File: user/main.go [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+	"github.com/aws/aws-lambda-go/lambda"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+	"github.com/silinternational/speed-snitch-admin-api"
+	"github.com/silinternational/speed-snitch-admin-api/db"
+	"net/http"
+)
+
+var dynamo = dynamodb.New(session.New(), aws.NewConfig().WithRegion("us-east-1"))
```
Comment (fillup) [inline user/main.go]: unused?

File: version/main.go [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+	"github.com/aws/aws-lambda-go/lambda"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+	"github.com/silinternational/speed-snitch-admin-api"
+	"github.com/silinternational/speed-snitch-admin-api/db"
+	"net/http"
+)
+
+var dynamo = dynamodb.New(session.New(), aws.NewConfig().WithRegion("us-east-1"))
```
Comment (fillup) [inline version/main.go]: unused?

File: domain_test.go (hunk 1 above)
Comment (Baggerone) [reply domain_test.go]: Done.

File: node/main.go (hunk 2 above)
Comment (Baggerone) [reply node/main.go]: Removed.

File: user/main.go (hunk 3 above)
Comment (Baggerone) [reply user/main.go]: Removed.

File: version/main.go (hunk 4 above)
Comment (Baggerone) [reply version/main.go]: Removed.

**Pull request merged**